tests/issues/issue226.icry.stdout

```diff
@@ -16,6 +16,35 @@ Constraint Synonyms
     type constraint i <= j = j >= i
     type constraint i > j = i >= 1 + j
 
+Primitive Types
```

Comment:
We're printing the kind of consraints as `Prop` here, but we're using the `@` symbol in the prelude.  We should pick one.

Reply:
Good point, I forgot that we do actually print kinds.  Maybe `Prop` makes more sense.